backend/tracim_backend/__init__.py

```diff
@@ -254,7 +254,14 @@ def web(global_config: OrderedDict, **local_settings) -> Router:
     context = TracimPyramidContext(
         configurator=configurator, default_error_builder=ErrorSchema(), debug=app_config.DEBUG
     )
-    hapic.set_context(context)
+    # HACK - G.M - 2021-07-01 - Force reset of context if already set.
+    # This case happened in some test now for unclear reasons.
```

Comment:
Do we plan to / is there a way get rid of this hack? If so, maybe write an issue, if not, you can simply resolve this conversation.

Reply:
I suspect it's a case that happened only in command test as we go multiple time in this code. but i'm not really sure.